crates/cxx-qt-gen/src/naming/name.rs

```diff
@@ -88,4 +93,8 @@ impl Name {
             module: module.clone(),
         })
     }
+
+    pub(super) fn cxx_unqualified(&self) -> String {
```

Comment:
doc line what this is and used for would be useful

Reply:
Done